web_src/js/features/repo-projects.ts

```diff
@@ -134,55 +134,47 @@ export function initRepoProject() {
             divider.style.removeProperty('color');
           }
         }
-        $('.ui.modal').modal('hide');
+        fomanticQuery('.ui.modal').modal('hide');
       }
     });
   }
 
-  $('.default-project-column-modal').each(function () {
-    const $boardColumn = $(this).closest('.project-column');
-    const $showButton = $($boardColumn).find('.default-project-column-show');
-    const $commitButton = $(this).find('.actions > .ok.button');
-
-    $($commitButton).on('click', async (e) => {
+  for (const modal of document.querySelectorAll('.default-project-column-modal')) {
+    const column = modal.closest('.project-column');
+    const showBtn = column.querySelector('.default-project-column-show');
+    const okBtn = modal.querySelector('.actions .ok.button');
+    okBtn.addEventListener('click', async (e: MouseEvent) => {
       e.preventDefault();
-
       try {
-        await POST($($showButton).data('url'));
+        await POST(showBtn.getAttribute('data-url'));
```

Comment:
These lines could be completely removed and replaces by `link-action` or `show-modal`